src/test/regress/expected/multi_create_shards.out

```diff
@@ -48,8 +48,10 @@ DETAIL:  Distributed relations must not specify the WITH (OIDS) option in their
 ALTER TABLE table_to_distribute SET WITHOUT OIDS;
 -- use an index instead of table name
 SELECT master_create_distributed_table('table_to_distribute_pkey', 'id', 'hash');
-ERROR:  cannot distribute relation: table_to_distribute_pkey
-DETAIL:  Distributed relations must be regular or foreign tables.
+ERROR:  "table_to_distribute_pkey" is an index
+LINE 1: SELECT EXISTS (SELECT 1 FROM public.table_to_distribute_pkey...
+                                     ^
+QUERY:  SELECT EXISTS (SELECT 1 FROM public.table_to_distribute_pkey)
```

Comment:
I suppose it might be slightly nicer if we do something like a `EnsureTableId(relationId);` first.